src/Components/Posts.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { InstagramEmbed } from "react-social-media-embed";
+import "../CSS/Posts.css";
+
+const postsData = [
```

Comment:
I don't love that these are hard-coded, but I also don't really feel like integrating with the instagram API so I'll let it slide...